content/cybersecurity/concepts/risk/risk.md

```diff
@@ -0,0 +1,13 @@
+---
+Title: 'Risk' # Required; the file name should be the same as the title, but lowercase, with dashes instead of spaces, and all punctuation removed
+Description: 'Risk is the possibility or likelihood that a threat will exploit a vulnerability to cause harm to an asset.' # Required; ideally under 150 characters and starts with a noun (used in search engine results and content previews)
+Subjects: # Please only use Subjects in the subjects.md file (https://github.com/Codecademy/docs/blob/main/documentation/subjects.md). If that list feels insufficient, feel free to create a new Subject and add it to subjects.md in your PR!
+  - 'Cybersecurity'
+Tags: # Please only use Tags in the tags.md file (https://github.com/Codecademy/docs/blob/main/documentation/tags.md). If that list feels insufficient, feel free to create a new Tag and add it to tags.md in your PR!
+  - 'Cybersecurity'
+CatalogContent: # Please use course/path landing page slugs, rather than linking to individual content items. If listing multiple items, please put the most relevant one first
+  - 'introduction-to-cybersecurity'
+  - 'paths/fundamentals-of-cybersecurity'
+---
+
+**Risk** is sometimes calculated as the product of threat and vulnerability.
```

Comment:
Can we have at least a paragraph here rather than this single sentence?  Can we incorporate the sentence from the description into here as well?